### ISSUE TITLE
Feature Request: User API to display active bindings

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#661]
I still have not memorized the Ginga bindings and sometimes wish to know what are the shortcuts I can actually use in a viewer. While there are documentation at https://ginga.readthedocs.io/en/latest/quickref.html , they are the default settings and not guaranteed to work as advertised if bindings are customized.

Programmatically, I could do something like this:
```python
>>> from collections import Iterable
>>> from astropy.table import Table
>>> from ginga.web.jupyterw.ImageViewJpw import EnhancedCanvasView
>>> viewer = EnhancedCanvasView()
>>> bd = viewer.get_bindings()
>>> t = Table(names=('action', 'key(s)'), dtype=('S128', 'S255'))
>>> for k in bd.settings.group:
...    v = bd.settings.group[k]
...    val = v.get()
...    if val is None or not isinstance(val, Iterable):
...        continue
...    col2 = ', '.join([s for s in val if s is not None])
...    t.add_row((k, col2))
>>> print(t)
       action              key(s)       
------------------- --------------------
          mod_shift     shift_l, shift_r
           mod_ctrl control_l, control_r
            mod_win           meta_right
          dmod_meta                space
          dmod_draw                  __b
          dmod_cmap                  __y
          dmod_cuts                  __s
          dmod_dist                  __d
      dmod_contrast                  __t
        dmod_rotate                  __r
                ...                  ...
    ms_camera_orbit          camera+left
ms_camera_pan_delta         camera+right
           ms_naxis           naxis+left
            pi_zoom                pinch
     pi_zoom_origin          shift+pinch
             pa_pan                  pan
            pa_zoom          freepan+pan
     pa_zoom_origin    freepan+shift+pan
           pa_naxis            naxis+pan
      pinch_actions                 zoom
Length = 124 rows
```

But even this is not completely user friendly, as I need to know, say,  what `mod_shift` means in plain English. Is there a simpler way to find out?

**Comment by ejeschke** [comment]
Examining the bindings is the most reliable way to find out all the user interface mappings.   Examining the settings of this object will yield the mini language fragments, from which you can deduce the behaviors.

In your "action" column the prefix ('mod_', 'dmod_', 'ms_', etc) determines what is being defined.  Here is a rough summary:
* mod: defining the modifier keys
* dmod: defining a mode
* ms: defining a mouse action (btn down, drag, up)
* kp: defining a keypress action
* sc: defining a scrolling action (mouse wheel or some trackpads)
* pi: pinch
* pa: pan
* etc.

The action name defines the method name within the bindings that will be the callback for that action (e.g. kb_pan_set).

In the column labeled "key(s)" is the list of bindings (these are not always keys, they can be mouse buttons or gestures, for example).  Each binding in the list shows the combination of things that will trigger that action.  From the example shown above, pressing down the "shift" modifier (defined to be either shift left or shift right, at the top) while performing the pinch gesture will invoke `pi_zoom_origin`.

The format for a binding is [MODE+][MODIFIER+]TRIGGER.

So if you wanted to show all the key bindings, you could filter the list for all the `kp_` actions and then show the bindings for them.
